src/test/java/com/virtuslab/archunit/ClassStructureTestSuite.java

```diff
@@ -99,6 +101,7 @@ public void all_classes_should_be_referenced() {
             // doesn't see accesses to static fields
             "com.virtuslab.gitmachete.frontend.defs")
         .and().doNotBelongToAnyOf(classesReferencedFromPluginXmlAttributes)
+        .and().doNotHaveFullyQualifiedName(SlideInRootBackgroundable.class.getName())
```

Comment:
To be removed on https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1168